documentation/docs/guide/setting/Using-BCGov-Block-Theme.md

```diff
@@ -19,6 +19,40 @@ Some features will need to be turned on to become available to the admin environ
 
 Other settings for admin enabling options in the Block Theme can be found on the  **BCGov Block Theme Settings**  page
 
+### Custom Notice Banner
+
+The Custom Notice Banner feature utilizes a checkbox to activate the Custom Notice Banner tool on the website. Enabling this tool allows the addition of a customizable banner to the site, offering various features such as the option to display the banner exclusively on the homepage. Users can further personalize the banner by selecting from a range of color options and accessing additional content settings. This feature proves valuable for prominently conveying important messages, promotions, or announcements to visitors, enhancing the overall user experience with a visually appealing and informative element.
+
+Once enabled the admin panel for it can be found under the admin menu Appearance > Custom Notice Banner.
+
+### Patterns
```

Comment:
should this be part of this documentation https://bcgov.github.io/bcgov-wordpress-block-theme/guide/content/Block-Patterns.html, is it the same or does it add different context

Reply:
As with the previous comment, would the addition of a link be acceptable?

<img width="782" alt="image" src="https://github.com/bcgov/bcgov-wordpress-block-theme/assets/5200460/4d0e3a11-2aad-4944-94f7-f0e22ccc73dd">